README.md

```diff
@@ -41,21 +56,40 @@ differences between alternative implementations by defining:
 The following table shows the TVMs currently defined for RISC-V. All of these
 TVMs only support a single hardware thread.
 
-TVM Name | Description
---- | ---
-`rv32ui` | RV32 user-level, integer only
-`rv32ua` | RV32 user-level, atomic
-`rv32uc` | RV32 user-level, compressed
-`rv32ud` | RV32 user-level, double-precision
-`rv32uf` | RV32 user-level, floating-point
-`rv32um` | RV32 user-level, integer mul/div
-`rv32uxpulpimg` | RV32 user-level, Xpulpimg extension
-`rv32si` | RV32 supervisor-level, integer only
-`rv64ui` | RV64 user-level, integer only
-`rv64uf` | RV64 user-level, integer and floating-point
-`rv64uv` | RV64 user-level, integer, floating-point, and vector
-`rv64si` | RV64 supervisor-level, integer only
-`rv64sv` | RV64 supervisor-level, integer and vector
+| TVM Name                    | Description                                           |
+| --------------------------- | ------------------------------------------------------|
+| `rv32ui`                    | RV32 user-level, integer only                         |
+| `rv32ua`                    | RV32 user-level, atomic                               |
+| `rv32uc`                    | RV32 user-level, compressed                           |
+| `rv32ud`                    | RV32 user-level, double-precision                     |
+| `rv32uf`                    | RV32 user-level, floating-point                       |
+| `rv32um`                    | RV32 user-level, integer mul/div                      |
+| `rv32uxpulpabs`             | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpbitop`           | RV32 user-level, Xpulp extension subset (incomplete)  |
+| `rv32uxpulpbitopsmall`      | RV32 user-level, Xpulp extension subset (incomplete)  |
+| `rv32uxpulpbr`              | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpclip`            | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpmacsi`           | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpminmax`          | RV32 user-level, Xpulp extension subset (incomplete)  |
+| `rv32uxpulppostmod`         | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpslet`            | RV32 user-level, Xpulp extension subset (complete)    |
+| `rv32uxpulpvect`            | RV32 user-level, Xpulp extension subset (incomplete)  |
+| `rv32uxpulpvectshufflepack` | RV32 user-level, Xpulp extension subset (incomplete)  |
+| WIP                         | complete and add further RV32 Xpulp extension subsets |
```

Comment:
Great.
Maybe you don't need the label `(complete)`. I guess you can just signal the incomplete ones; make sure to explain after the table what you mean by `(incomplete)` i.e. that not all instructions are provided with tests by this suite.